_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
+   3. Non-complex and not core to mandate IT Solutions may be designed and delivered by third party integrators.
+3. IT Solutions, whether built internally or built by a third party and delivered as a set of pre-packaged COTS to configure and/or as software as a service, adhere to the Architectural principles as listed in section [Architecture](#architecture)
+4. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
+   - Accessibility test results
+   - Integration testing results
+   - Security test results
+   - API consumption statistics
+   - Production deployment audit traces
+   - Availability statistics
+5. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
+6. Accessibility Assessment uses the Target-Accessibility methodology
+7. Audit Assessment uses the Target-Audit methodology
+8. IT Solutions reuse existing IT Products, including:
+    - Internally developed software
+    - Open Source Software (internally and externally developed)
+    - Third party Services (SaaS, PaaS, and IaaS) and COTS
+
+### Finance
+
+1. IT Solutions sizes are minimized as much as possible
+2. IT Solutions leverage existing, reusable, scalable and approved IT Products to minimize cost when possible
+   - IT Products requiring purchased licences are not deemed as reusable or scalable for the purpose of this criterion.
+3. IT Solutions, once in production, have dedicated and recurring budgets to support their evolution over their lifecycle, and are managed in tandem by the primary business owner and the dedicated IT Products teams.
+   - The IT Products teams are responsible for maintaining their own products as part of the IT Solutions. (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html))
+   - A same team managing an IT Product may provide service to multiple IT Solutions; IT Products are thus required to have a scalable and easily adaptable financial model supporting multiple business owners if required (similar to SaaS pay as you use or yearly subscriptions per users, etc.).
+
+### Architecture
+
+1. IT Solutions are aligned with the Digital Standards and the Digital Operations Strategic Plan. As such, IT Solutions are built or bought using IT Products that meet the following criteria:
+   - Designed with users
+   - Iterative and improved frequently
+   - Uses Open Standards and Solutions
+   - Open by default
+   - Designed, developed, delivered and maintained by widely collaborating, both internally and externally
+2. IT Solutions are also required to be designed with a Service Oriented Architecture approach so that its components (IT Products) meet the following criteria:
+   - Interoperable
+   - Reusable
+   - Scalable
+   - Interchangeable
+3. IT Solutions are to leverage Cloud as a preferred delivery option for its IT Products
+
+## Coherent set of actions
```

Comment:
Based on the recommended comments above, there should be 5:

- produce a EARB standard definition of IT Solutions (IT Solution - IT Product - Software+configuration - Infrastructure+configuration). Stakeholders involved would be Architecture teams (Business Arch, EA, Solution Arc, Technical Arc), Senior Advisors, ARC, Technical Standards Working Group (TSWG), and ultimately EARB

- Produce an IT Solution risk assessment framework (how to quantify risks for the scope of an IT Solution). Stakeholders involved would be same as above with the addition of CDO (because of the data involved in a vendor locking risk assessment). And especially Business Architecture because the risk assessment must roll up to the Business Capability Model (in my mind).

- Produce a vendor locking risk assessment framework. Different than the IT Solution risk assessment framework  because the vendor locking risk assessment framework is at the IT Product level, not the IT solution level. Vendor locking greatly affects data (e.g. if we change our mind and go with another vendor, what happens to the data that is stored on our first vendor's infastructure? Do we loose it? Do we have to migrate it (which is tremendously costly and error prone)? Stakeholders involved are Architecture teams and CDO.

- produce a software legal terms guidance document ( guidance on OSS adoption, Cloud subscriptions, shared-risk and responsibility). Stakeholders to include are legal branch, Cloud CoE (they have a lot of knowledge in this area), CFOB IPPM, and even probably TBS Procurement Policy

- Endorsement of ESDC OSS framework

Reply:
Shouldn't we include Ops at some point in these though?